packages/mdc-animation/README.md

```diff
@@ -27,59 +27,68 @@ npm install --save @material/animation
 
 ## Usage
 
-We currently have variables for the following 4 animation curves:
+### CSS Classes
+
+Some components have a set typographic style. For example, a raised MDC Checkbox enter and exit curves.
+
+If you want to animate an element, which is not a Material Design component, you can apply the following CSS classes.
 
-| Variable name | timing function | use case |
-| --- | --- | --- |
-| `$mdc-animation-standard-curve-timing-function` | `cubic-bezier(.4, 0, .2, 1)` | Standard curve; any animations that are visible from start to finish (e.g. a FAB transforming into a toolbar) |
-| `$mdc-animation-deceleration-curve-timing-function` | `cubic-bezier(0, 0, .2, 1)` | Animations that cause objects to enter the screen (e.g. a fade in) |
-| `$mdc-animation-acceleration-curve-timing-function` | `cubic-bezier(.4, 0, 1, 1)` | Animations that cause objects to leave the screen permanently (e.g. a fade out) |
-| `$mdc-animation-sharp-curve-timing-function` | `cubic-bezier(.4, 0, .6, 1)` | Animations that cause objects to leave the screen temporarily (e.g. closing a drawer) |
+CSS Class | Description
+--- | ---
+`mdc-animation-deceleration-curve` | Sets the speed curve to deceleration
```

Comment:
I don't want to put `animation-timing-function` in the description...because it is an implementation detail. Is there any other way to make this more clear without just documenting the output of the CSS class?